### PR TITLE
Make changes to accommondate broker4j getAccounts

### PR DESCRIPTION
**Pull request by rpdome**
https://github.com/AzureAD/ad-accounts-for-android/pull/1690

### CODE REVIEW

**Review by shahzaibj — APPROVED** [review]

File: common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java [hunk 1]
```diff
@@ -60,6 +60,14 @@
     @NonNull
     IDevicePopManager getDefaultDevicePopManager() throws ClientException;
 
+    /**
+     * Gets a {@link IDevicePopManager} associated to the alias.
+     *
+     * @throws ClientException if it fails to initalize, or if the operation is not supported by the platform.
+     */
+    @NonNull
+    IDevicePopManager getDevicePopManager(@Nullable final String alias) throws ClientException;
```
Comment (AdamBJohnsonx) [inline common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java:69]: We probably don't want an IDevicePopManager here.

Comment (rpdome) [reply common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java:69]: 1. I moved it up from below.
2. If you meant this class doesn't belong to this object, you may be right. That said, we decided that we won't be touching the PoP layer in P0. Once we are ready to support PoP in Linux broker, we'll come back here and actually figure out what, in this class, actually belongs to IPlatformComponents.

Comment (AdamBJohnsonx) [reply common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java:69]: I just didn't find a usage.

Comment (AdamBJohnsonx) [reply common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java:69]: And, OK, I see it now.  Which is why I didn't see the usage.  Nevermind. :headdesk

**Review by AdamBJohnsonx — APPROVED** [review]

**Pull request merged**